.github/workflows/lint.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Install Rust
         uses: dtolnay/rust-toolchain@stable
         with:
-          toolchain: 1.63
+          toolchain: 1.64
```

Comment:
Only the lint pass needs an upgrade to rust 1.64?

Reply:
Actually, the new webc needs 1.64.

Do we know why that is, @fschutt ?

But if our policy is current + last 2 versions then bumping to 1.64 would be fine, sine 1.66 just released.

Reply:
@theduke as far as I know, it needs 1.64 for inheriting cargo metadata, but you'd need to ask @Michael-F-Bryan about this, he did the version bumping

Reply:
My issue here is that just lint.yaml is bumped, not any other yaml that are stying in 1.63
This is inconsistant, so it's either not needed on linter or missing everywhere else.

Reply:
Oh, well, the `rust-toolchain` file also has the bump!

Reply:
Ok, I pused a commit to bumped the rust toolchain to 1.64 for all CI step then.